specmark.md

```diff
@@ -0,0 +1,353 @@
+and is therefore defined by an SPDX data creator.
+
+## Metadata
+
+- name: SimpleLicensingText
+- SubclassOf: /Core/Element
+- Instantiability: Concrete
+
+## Properties
+
+- licenseText
+  - type: xsd:string
+  - minCount: 1
+  - maxCount: 1
+```
+
+will give this RDF graph
+(in [Turtle syntax](https://en.wikipedia.org/wiki/Turtle_(syntax))):
+
+```ttl
+<https://spdx.org/rdf/3.0.1/terms/SimpleLicensing/SimpleLicensingText> a owl:Class,
+        sh:NodeShape ;
+    rdfs:comment "A license or addition that is not listed on the SPDX License List."@en ;
+    rdfs:subClassOf <https://spdx.org/rdf/3.0.1/terms/Core/Element> ;
+    sh:nodeKind sh:IRI ;
+    sh:property [ sh:datatype xsd:string ;
+            sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:nodeKind sh:Literal ;
+            sh:path <https://spdx.org/rdf/3.0.1/terms/SimpleLicensing/licenseText> ] .
+```
+
+## Syntax
+
+### Classes
+
+Allowed headings:
+
+- Summary
+- Description
+- Metadata
+  - name: \<class_name\>
+  - SubclassOf: \<class_name\> OR "none" *(Must have one if Instantiability is "Concrete")*
+  - Instantiability: "Abstract" OR "Concrete" *(Default: Concrete)*
+- Properties *(Optional)*
+  - \<property_name\>
+    - type: \<datatype_name\>
+    - minCount: \<number\> *(Optional)*
+    - maxCount: \<number\> *(Optional)*
+  - ...
+- External properties restrictions *(Optional)*
+  - \<external_property_name\>
+    - minCount: \<number\> *(Optional)*
+    - maxCount: \<number\> *(Optional)*
+  - ...
+
+#### Class example
+
+```markdown
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Bot
+
+## Summary
+
+An automated agent.
+
+## Description
+
+A class example.
+
+## Metadata
+
+- name: Bot
+- SubclassOf: Agent
+- Instantiability: Concrete
+
+## Properties
+
+- prohibitedTask
+  - type: xsd:string
+  - minCount: 1
+
+## External properties restrictions
+
+- /Core/Element/name
+  - minCount: 1
+```
+
+### Datatypes
+
+Allowed headings:
+
+- Summary
+- Description
+- Metadata
+  - name: \<datatype_name\>
+  - SubclassOf: \<class_name\>
+- Format
+  - pattern: \<regular_expression\>
+
+#### Datatype example
+
+```markdown
+SPDX-License-Identifier: Community-Spec-1.0
+
+# DateTime
+
+## Summary
+
+A string representing a specific date and time.
+
+## Description
+
+A DateTime is a string representation of a specific date and time.
+
+## Metadata
+
+- name: DateTime
+- SubclassOf: xsd:dateTimeStamp
+
+## Format
+
+- pattern: ^\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\dZ$
+```
+
+### Individuals
+
+Allowed headings:
+
+- Summary
+- Description
+- Metadata
+  - name: \<individual_name\>
+  - type:  \<class_name\>
+- Property Values
+  - \<property_name\>: \<property_value\>
+  - ...
+
+#### Individual example
+
+```markdown
+SPDX-License-Identifier: Community-Spec-1.0
+
+# NoneElement
+
+## Summary
+
+An Individual Value example.
+
+## Description
+
+Blah.
+
+## Metadata
+
+- name: NoneElement
+- type: Element
+
+## Property Values
+
+- name: "NONE"
```

Comment:
Individuals may define an IRI.

I am not sure "name" is used for anything (or processed at all)

Reply:
Thank you. IRI added to Metadata.

--

For the "name" property, I took the example from [/Core/NoneElement](https://github.com/spdx/spdx-3-model/blob/main/model/Core/Individuals/NoneElement.md).

Not sure about their original intent. 

These individuals have name as "NONE":
- [/Core/NoneElement](https://github.com/spdx/spdx-3-model/blob/main/model/Core/Individuals/NoneElement.md)
- [/ExpandedLicensing/NoneLicense](https://github.com/spdx/spdx-3-model/blob/main/model/ExpandedLicensing/Individuals/NoneLicense.md)

These two have name as "NOASSERTION":
- [/Core/NoAssertionElement](https://github.com/spdx/spdx-3-model/blob/main/model/Core/Individuals/NoAssertionElement.md)
- [/ExpandedLicensing/NoAssertionLicense](https://github.com/spdx/spdx-3-model/blob/main/model/ExpandedLicensing/Individuals/NoAssertionLicense.md)

- As a label for human use? -- in [Relationship description](https://github.com/spdx/spdx-3-model/blob/main/model/Core/Classes/Relationship.md), "NONE" and "NOASSERTION" are used as labels to refer to `/Core/NoneElement` and `/Core/NoAssertionElement` individuals.
- As a comparable string for machine use? -- in [copyrightText](https://github.com/spdx/spdx-3-model/blob/main/model/Software/Properties/copyrightText.md), it describes the convention of using "NONE" and "NOASSERTION" string values. It may be possible to compare copyrightText value with the names of ExpandedLicensing individuals?